tests/test_various.rs

```diff
@@ -133,8 +133,10 @@ impl PickleSupport {
     }
 }
 
-fn add_module(py: Python<'_>, module: &PyModule) -> PyResult<()> {
-    py.import_bound("sys")?
+fn add_module(module: Bound<'_, PyModule>) -> PyResult<()> {
+    module
+        .py()
+        .import_bound("sys")?
```

Comment:
Any reason not to just use `module.import_bound`?

Reply:
None at all, good idea to simplify.